frontend/src/app/components/pull-request/pull-request.component.html

```diff
@@ -8,10 +8,19 @@
     </div>
     <a [routerLink]="['/projects']"><button>Select another project</button></a>
   </div>
-  <div *ngIf="!pullRequest">
+  <div *ngIf="!pullRequest && message">
+    <div>
+      <label>{{ message }} Pull request not updated.</label>
+    </div>
+    <a [routerLink]="['/projects']"><button>Select another project</button></a>
+  </div>
+  <div *ngIf="!pullRequest && !message">
     <div class="spinner">
       <fa-icon icon="spinner" spin="true" size="5x"></fa-icon>
     </div>
-    <div class="message">Please wait while we create the pull request on your behalf...</div>
+    <div class="message">Please wait while we
+      <span *ngIf="updatePullRequest">update</span>
+      <span *ngIf="!updatePullRequest">create</span> the pull request on your behalf...
+    </div>
```

Comment:
For future i18n, easier if we don't do too much grammatical assembly.

```suggestion
    <div class="message" *ngIf="updatePullRequest">Please wait while we update the pull request on your behalf...</div>
    <div class="message" *ngIf="!updatePullRequest">Please wait while we create the pull request on your behalf...</div>
```

Reply:
Good idea. Makes it easier to read.